internal/provider/azure/storage.go

```diff
@@ -640,7 +645,7 @@ func (v *azureVolumeSource) updateVirtualMachines(
 			_, err = poller.PollUntilDone(ctx, nil)
 		}
 		if err != nil {
-			if errorutils.MaybeInvalidateCredential(err, ctx) {
+			if handled, _ := errorutils.HandleCredentialError(ctx, v.env.credentialInvalidator, err); handled {
```

Comment:
`handled` isn't very clear imo. To me it suggests the error was handled so we don't need to worry about it, which I don't think is what this means.

Previously I think `invalidated` was used?

Reply:
Agreed.